public/clientAppId.jsonld

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
See [here](https://docs.inrupt.com/developer-tools/javascript/client-libraries/tutorial/authenticate-client/) for more information on this document